samwise.js

```diff
@@ -2,58 +2,61 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!